instructionAPI/syscall-x86.cpp

```diff
@@ -111,6 +122,10 @@ int main(int argc, char **argv) {
     return -1;
   }
 
+  // Convention for CTest
+  constexpr int PASS =  0;
+  constexpr int FAIL = -1;
```

Comment:
The return value from an executable is 0 to 255 (the lower byte of the int is the value returned), so `1` is the common error value.

```suggestion
  constexpr int FAIL = 1;
```

Reply:
Fixed. I updated the other fail paths, as well.